feat: add default support for sphinx-llm (#590)
canonical/sphinx-docs-starter-pack: docs/how-to/contributing-myst.md

---
myst:
  html_meta:

>     description: Contribute to the documentation project with code, writing, or testing.

Reviewer: ```suggestion
    description: How to contribute code, documentation, and testing to the Starter Pack.
```

Author: https://github.com/canonical/sphinx-docs-starter-pack/pull/590/commits/47f855515d8142df3c7341d701d0d9e8c544aecc
